src/Microsoft.HttpRepl/Commands/ClearCommand.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.HttpRepl.Commands
 {
     public class ClearCommand : ICommand<object, ICoreParseResult>
     {
-        private static readonly string Name = "clear";
+        public string Name => "clear";
```

Comment:
nit: arrange this with the other public members instead of with the private field?